File: Server/TruckInfoModule/Models/TransportInfo.cs
using System;
using System.Collections.Generic;
using System.Text;
using TransportInfoModule.Models;

namespace TransportInfoModule.Models
{
    public class TransportInfo
    {
        public TrailerDamage? TrailerDamage { get; set; } = null;
        public TruckDamage TruckDamage { get; set; } = new();
        public FuelInfo FuelInfo { get; set; } = new();
        public List<ErrorDto> Errors { get; set; } = new();
    }
}

File: Client/app/src/main/java/com/nks/interactive/multimediapanel/models/transportInfo/TransportInfo.kt
package com.nks.interactive.multimediapanel.models.transportInfo

data class TransportInfo(
    val trailerDamage: TrailerDamage? = null,
    val truckDamage: TruckDamage = TruckDamage(),
    val fuelInfo: FuelInfo = FuelInfo(),
    val errors: List<ErrorDto> = emptyList()
)